scripts/start.js

```diff
@@ -62,11 +62,11 @@ function runElectronApp(port) {
   electronProcess.stdout.on('data', data => {
     // dont log blank output or empty newlines
     const output = data.toString().trim();
-    if (output.length) console.log(chalk.green('[ELECTRON]'), output);
+    if (output.length) console.info(chalk.green('[ELECTRON]'), output);
   });
   electronProcess.stderr.on('data', data => {
     const output = data.toString();
-    console.log(chalk.red(`[ELECTRON] ${output}`));
+    console.error(chalk.red(`[ELECTRON] ${output}`));
```

Comment:
I swear I did this change a couple weeks ago o_O maybe a merge accidentally erased it?